lib/ae_canary/fork_monitor/model/alert.ex

```diff
@@ -0,0 +1,134 @@
+defmodule AeCanary.ForkMonitor.Model.Alert do
+  require Logger
+
+  @minumum_fork_length 2
```

Comment:
This and the following line are good subjects for configuration, maybe not in this PR.